tests/RealMeServiceTest.php

```diff
@@ -25,7 +34,7 @@ public function testGetCertificateContents()
         file_put_contents($path, $contents);
```

Comment:
There's a couple of parts in this test that won't work in SS4:

* Assets aren't always stored in the assets folder any more. You can use `TestAssetStore` to let you use a path in the assets folder though.
* `file_get_contents(BASE_PATH . '/realme/tests` won't work any more because the module is now a vendor module, and when Travis runs these tests it's installed as the project, not a module. Instead you could use something like `dirname(__FILE__) . '/certs/...` so it's relative. Note that this would likely tie into how you implement the TestAssetStore as well.

Reply:
:+1: